metadata-ingestion/src/datahub/ingestion/source/gc/dataprocess_cleanup.py

```diff
@@ -226,10 +229,12 @@ def fetch_dpis(self, job_urn: str, batch_size: int) -> List[dict]:
                 runs = runs_data.get("runs")
                 dpis.extend(runs)
                 start += batch_size
-                if len(runs) < batch_size:
+                if len(runs) < batch_size or start >= max_item:
```

Comment:
we probably should move to using scroll instead of search

Reply:
Unfortunately there is no scroll endpoint for this :(